qemu/tests/cfg/max_lun.cfg

```diff
@@ -0,0 +1,24 @@
+- max_lun:
+    virt_test_type = qemu
+    type = max_channel_lun
+    restart_vm = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
+    start_vm = no
+    only ppc64le ppc64
+    only Linux
+    spapr_vscsi_reg = 0x71000001
+    stg_image_name = "images/%s"
+    force_create_image_image1 = no
+    force_create_image = yes
+    channel = "channel=0"
+    bootindex_image1 = 0
+    error_info = "bad scsi device lun: 32"
+    variants luns:
+        - lun_32:
+            stg_image_num = 32
+        - lun_33:
+            stg_image_num = 32
+            image_name_stg32 = images/stg32
+            image_format_stg32 = qcow2
+            extra_params = "-blockdev node-name=file_stg32,driver=file,auto-read-only=on,discard=unmap,aio=threads,filename=/home/kar/vt_test_images/stg32.qcow2,cache.direct=on,cache.no-flush=off -blockdev node-name=drive_stg32,driver=qcow2,read-only=off,cache.direct=on,cache.no-flush=off,file=file_stg32 -device scsi-hd,lun=32,id=stg32,bus=spapr_vscsi0.0,drive=drive_stg32,write-cache=on,channel=0" 
```

Comment:
Since `data_dir` is a relative path, it can be `/home/kar/vt_test_images/` or `/var/lib/avocado/data/avocado-vt/images/`, so suggest move this line into py file under `if params['luns'] == "lun_33":`, you can define `extra_params` with `image.image_filename`.
Although using `spapr_vscsi0.0` directly is not the best choice, for a negative test, it's okay for me.

Reply:
BTW, if mandatory use `blockdev`, please limit the qemu version to [3.0.0, )